### PR TITLE
chore: add notes about leader-election

### PR DESCRIPTION
**Pull request by iguoyr**
Signed-off-by: SiyuChen <ryougi201@gmail.com>

close https://github.com/chaos-mesh/website/issues/139

### CODE REVIEW

**Comment by iguoyr** [comment]
/cc @cwen0 @STRRL 
PTAL

File: docs/production-installation-using-helm.md [hunk 1]
```diff
@@ -97,6 +97,12 @@ To install Chaos Mesh of a specific version, add the `--version xxx` parameter a
 
 :::
 
+:::note
+
+In order to ensure high availability, Chaos Mesh turn on `leader-election` feature by default, if you do not need this feature, you can manually turn it off by `--set controllerManager.leaderElection.enabled=false`.
```
Comment (en-jin19) [inline docs/production-installation-using-helm.md:102]:
```suggestion
To ensure high availability, Chaos Mesh turns on `leader-election` feature by default. If you do not need to use this feature, you can manually turn it off through `--set controllerManager.leaderElection.enabled=false`.
```

File: i18n/zh/docusaurus-plugin-content-docs/current/common/verify-installation.md [hunk 2]
```diff
@@ -25,3 +25,16 @@ kubectl describe po -n chaos-testing chaos-controller-manager-69fd5c46c8-xlqpc
 ```
 
 :::
+
+:::note 注意
+
+如果手动关闭了 `leader-election`，那么 `chaos-controller-manager` 应该只有1个实例。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/common/verify-installation.md:31]:
```suggestion
如果手动关闭了 `leader-election`，`chaos-controller-manager` 应该只有 1 个实例。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/production-installation-using-helm.md [hunk 3]
```diff
@@ -101,6 +101,12 @@ helm install chaos-mesh chaos-mesh/chaos-mesh -n=chaos-testing --set chaosDaemon
 
 :::
 
+:::note 注意
+
+为了保证高可用，Chaos Mesh 默认开启了 `leader-election` 特性，如果你不需要这个特性，可以通过 `--set controllerManager.leaderElection.enabled=false` 来手动关闭。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/production-installation-using-helm.md:106]:
```suggestion
为了保证高可用性，Chaos Mesh 默认开启了 `leader-election` 特性。如果不需要使用这个特性，请通过 `--set controllerManager.leaderElection.enabled=false` 手动关闭该特性。
```

File: docs/production-installation-using-helm.md (hunk 1 above)
Comment (iguoyr) [reply docs/production-installation-using-helm.md]: fixed!

File: i18n/zh/docusaurus-plugin-content-docs/current/common/verify-installation.md (hunk 2 above)
Comment (iguoyr) [reply i18n/zh/docusaurus-plugin-content-docs/current/common/verify-installation.md]: fixed!

File: i18n/zh/docusaurus-plugin-content-docs/current/production-installation-using-helm.md (hunk 3 above)
Comment (iguoyr) [reply i18n/zh/docusaurus-plugin-content-docs/current/production-installation-using-helm.md]: fixed!

**Review by en-jin19 — APPROVED** [review]
LGTM

**Comment by en-jin19** [comment]
@cwen0 @STRRL Could you please take a look and approve this PR if there is no problem? Thanks!

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**